Code/Tools/Player/CMakeLists.txt

```diff
@@ -14,6 +14,28 @@ target_link_libraries(${PROJECT_NAME}
   GameEngine
 )
 
+if (EZ_CMAKE_PLATFORM_WINDOWS_UWP)
+  # Due to app sandboxing we need to explicitly name required plugins for UWP.
+  target_link_libraries(${PROJECT_NAME}
```

Comment:
Could this be done through our finalize.cmake files? No high priority though. They are used to add a dependency to another project without the other project needing to know about it. Eg kraut can add a dependency to player, or any other custom plugin can do that, and player does not need to know about all active plugins.

Reply:
No unfortunately it can't. The finalizer is called in a different scope, calling find_package there does nothing to help resolve the generator expression at generation time. We need to call find_package at the scope of Code\Tools\Player\CMakeLists.txt so that the project itself can resolve the generator expression. At least I don't know how else I can fix that.